tests/distributed/test_pipeline_parallel.py

```diff
@@ -29,14 +29,11 @@
                              (1, 4, 1, 0, "meta-llama/Meta-Llama-3-8B", "mp"),
                          ])
 def test_compare_tp(TP_SIZE, PP_SIZE, EAGER_MODE, CHUNKED_PREFILL, MODEL_NAME,
-                    DIST_BACKEND):
+                    DIST_BACKEND, USE_RAY_ADAG, USE_RAY_ADAG_NCCL):
```

Comment:
I think you need to remove these two args from function signature.

Reply:
Totally.